x/bank/client/cli/cli_test.go

```diff
@@ -0,0 +1,191 @@
+package cli_test
+
+import (
+	//"context"
+	"fmt"
+
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/suite"
+
+	//tmcli "github.com/tendermint/tendermint/libs/cli"
+
+	//	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+
+	//clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	"github.com/cosmos/cosmos-sdk/testutil/network"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	//"github.com/cosmos/cosmos-sdk/types/query"
+	//"github.com/cosmos/cosmos-sdk/x/bank/client/cli"
+	banktestutil "github.com/cosmos/cosmos-sdk/x/bank/client/testutil"
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
```

Comment:
remove commented out code (never comment out code to remove it) and group imports from the same repo